examples/employee-app/Dockerfile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# 1st stage, build the app
+FROM maven:3.5.4-jdk-9 as build
+
+WORKDIR /helidon
+
+# Create a first layer to cache the "Maven World" in the local repository.
+# Incremental docker builds will always resume after that, unless you update
+# the pom
+ADD pom.xml .
+RUN mvn package -DskipTests
+
+# Do the Maven build!
+# Incremental docker builds will resume here when you change sources
+ADD src src
+RUN mvn package -DskipTests
+
+RUN echo "done!"
+
+# 2nd stage, build the runtime image
+FROM openjdk:8-jre-slim
+WORKDIR /helidon
+
+# Copy the binary built in the 1st stage
+COPY --from=build /helidon/target/employee-app.jar ./
+COPY --from=build /helidon/target/libs ./libs
+COPY --from=build /helidon/classes/logging.properties ./
+COPY --from=build /helidon/public ./public
```

Comment:
the content of public should already be in the jar file

Reply:
We removed COPY instructions about the properties and public directory, but the app doesn't work if I remove the lib directory.

Reply:
Yes that's expected, the JAR and the lib folder are required.